index.bs

```diff
@@ -4481,8 +4481,9 @@ In order to perform a [=registration ceremony=], the [=[RP]=] MUST proceed as fo
         - If [=None|no attestation=] was provided, verify that [=None=] attestation is acceptable under [=[RP]=] policy.
         - If [=self attestation=] was used, verify that [=self attestation=] is acceptable under [=[RP]=] policy.
         - Otherwise, use the X.509 certificates returned as the [=attestation trust path=] from the [=verification procedure=]
-            to verify that the attestation public key either correctly chains up to an acceptable root certificate, or is itself an acceptable certificate (i.e., it and the root certificate obtained in Step 20 may be the same).
+            to verify that the attestation public key either correctly chains up to an acceptable root certificate, or is itself an acceptable certificate (i.e., it and the root certificate obtained in Step 19 may be the same).
 
+1. Check that the <code>[=credentialId=]</code> is &lt; 1024 bytes. Credential IDs larger than this bytes SHOULD cause the RP to fail this [=registration ceremony=].
```

Comment:
Nit: Typo?
```suggestion
1. Check that the <code>[=credentialId=]</code> is &lt; 1024 bytes. Credential IDs larger than this many bytes SHOULD cause the RP to fail this [=registration ceremony=].
```

(Or "this number of bytes"?)

Reply:
Thanks!